game-core/src/main/java/games/strategy/triplea/ResourceLoader.java

```diff
@@ -210,8 +209,9 @@ static String normalizeMapName(final String zipName) {
 
   private static List<File> getCandidatePaths(final String mapName) {
     final List<File> candidates = new ArrayList<>();
-    candidates.addAll(getMapDirectoryCandidates(mapName));
-    candidates.addAll(getMapZipFileCandidates(mapName));
+    candidates.addAll(
+        getMapDirectoryCandidates(mapName, ClientFileSystemHelper.getUserMapsFolder()));
+    candidates.addAll(getMapZipFileCandidates(mapName, ClientFileSystemHelper.getUserMapsFolder()));
```

Comment:
A question of taste, but if you want a more functional approach here, you could do
```java
Stream.of(
  getMapDirectoryCandidates(mapName, ClientFileSystemHelper.getUserMapsFolder()),
  getMapZipFileCandidates(mapName, ClientFileSystemHelper.getUserMapsFolder())
).flatMap(Collection::stream)
.collect(Collectors.toList());
```
or
Stream.concat(
  getMapDirectoryCandidates(mapName, ClientFileSystemHelper.getUserMapsFolder()).stream(),
  getMapZipFileCandidates(mapName, ClientFileSystemHelper.getUserMapsFolder()).stream()
).collect(Collectors.toList());
```